src/recipe.js

```diff
@@ -20,7 +20,34 @@ class Recipe {
     });
     return costCounter;
   }
-
+  returnInstructions() {
+    return this.instructions
+  }
+  filterByTag(letter) {
+    var tags =  this.tags.filter(tag =>{
+      return tag.split('').includes(letter.toLowerCase())
+    })
+    console.log(tags)
+    return tags
+  }
+  containsIngredient(inputIngredient) { 
```

Comment:
I like how you added a method here that isn't pointed out in the spec.  Good thinking ahead!